files/en-us/mozilla/firefox/releases/88/index.html

```diff
@@ -74,7 +74,11 @@ <h4 id="DOM">DOM</h4>
 
 <h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
 
-<p><em>No changes.</em></p>
+<ul>
+  <li>Marionette will no longer be enabled unless the <code>--marionette</code> command line argument or the <code>MOZ_MARIONETTE</code> environment variable have been specified. As such the <code>marionette.enabled</code> preference is no longer used. With this change the state of <code>navigator.webdriver</code> now correctly reflects the enabled state of Marionette ({{bug(1593343)}}).</li>
+  <li>Fixed a bug where pointer actions other than <code>down</code> and <code>up</code> inappropriatelly lead to buttons to be pressed ({{bug(1686361)}}).</li>
```

Comment:
```suggestion
  <li>Fixed a bug where pointer actions other than <code>down</code> and <code>up</code> inappropriately lead to buttons to be pressed ({{bug(1686361)}}).</li>
```

(and maybe "lead to buttons to be pressed" -> "lead (led?) buttons to be pressed")